src/network/networkpacket.cpp

```diff
@@ -63,7 +63,8 @@ void NetworkPacket::putRawPacket(u8 *data, u32 datasize, session_t peer_id)
 
 	// split command and datas
 	m_command = readU16(&data[0]);
-	memcpy(&m_data[0], &data[2], m_datasize);
+	if (m_datasize)
```

Comment:
please may you change this to   `if (m_datasize != 0)`   just to make it explicit that it's a numeric comparison rather than a boolean or pointer. It'll be compiled out anyway

Reply:
I found a better way of doing it - using m_data.data() instead of adding a condition check. The pull request should update soon..